pilot/cmd/pilot-agent/main.go

```diff
@@ -101,7 +101,7 @@ var (
 	istioNamespaceVar    = env.RegisterStringVar("ISTIO_NAMESPACE", "", "")
 	kubeAppProberNameVar = env.RegisterStringVar(status.KubeAppProberEnvName, "", "")
 	sdsEnabledVar        = env.RegisterBoolVar("SDS_ENABLED", false, "")
-	sdsUdsPathVar        = env.RegisterStringVar("SDS_UDS_PATH", "/var/run/sds/uds_path", "SDS unix domain socket path")
+	sdsUdsPathVar        = env.RegisterStringVar("SDS_UDS_PATH", "unix:/var/run/sds/uds_path", "SDS unix domain socket path")
```

Comment:
As I mentioned in the previous PR - UDS and unix: are redundant. 
If you want to send a URI ( which is not a bad idea ) - use SDS_ADDRESS. Or at least update the comment to indicate it's an URI that you are passing, not a path.

And I really can't see how detectSds which call stat() on the content of that env will work...

Reply:
The comment in this env var is updated.
detectSds checks sds address only if it is UDS address, checking of token path is not changed.